### DISCUSSION TITLE
The logo does not show when deployed by CI

### DISCUSSION THREAD

**Discussion by Kulezic101** [ortoniKC/ortoni-report#31] [General]
Hi,
The logo always seems to prepend the full path starting from root "/" to the logo file name.  If I just remove the full path using dev tools the logo shows.  How do I prevent the full path being pre-pended to the logo file?

**Comment by ortoniKC** [comment]
In the latest release it is fixed, 
Please keep your logo within the report folder.

**Comment by Kulezic101** [comment]
Thanks.  That seems to fix the logo issue.  But now the tests do not seem get executed properly for CI.  Running locally works.
The error that shows in the logs is:

Error: spawn xdg-open ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (node:internal/child_process:292:12)
    at onErrorNT (node:internal/child_process:484:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn xdg-open',
  path: 'xdg-open',
  spawnargs: [ 'http://localhost:2004/' ]

**Reply by ortoniKC** [reply]
As the ci has no ui, you're facing the issue. 
In the report config there is a new option as open: process.env.CI ? "never" : "always" 
This should fix.

**Reply by Kulezic101** [reply]
Like this?
const reportConfig: OrtoniReportConfig = {
  preferredTheme: 'dark',
  filename: 'index',
  authorName: '... Automation Tester',
  projectName: 'Ortoni Report V2.0.5',
  testType: 'Release',
  title: '... Automation Tester',
  logo: '..._logo.png',
  base64Image: true,
  folderPath: 'playwright-report',
  showProject: false,
  open: process.env.CI ? "never" : "always",
};

**Reply by ortoniKC** [reply]
Yes

**Comment by Kulezic101** [comment]
Thanks!  All is working now!
